utils/report_timing.py

```diff
@@ -0,0 +1,69 @@
+import argparse
```

Comment:
How is this different then the parse timing above?

Reply:
Those scripts are organized in the analogous way the resource usage parsing ones. `utils/lib/parse_timing.py` contains parsing routines while this one uses them.

This one is used in CI to catch any regressions to design implementation that would affect timing.